spec/inferno/test_runner_spec.rb

```diff
@@ -79,7 +79,11 @@ def error_results_message(error_results)
   end
 
   describe 'when running wait group' do
-    let(:group) { Inferno::Repositories::TestSuites.new.find('demo').groups.last }
+    let(:group) do
+      Inferno::Repositories::TestSuites.new.find('demo').groups.find do |group|
```

Comment:
`Inferno::Repositories::TestGroups.new.find('demo-wait_group')`

Same in `test_runs_spec.rb`